04-files.md

```diff
@@ -16,8 +16,8 @@ The only thing that's missing is a library with a rather unpleasant name:
 import glob
 ~~~
 
-The `glob` library contains a single function, also called `glob`,
```

Comment:
The `glob` library (or more precisely module) does not contain only the `glob()` function, but also the `iglob()` iterator and the `escape()` function, see https://docs.python.org/3/library/glob.html .